src/connect/ConnectWizard.ts

```diff
@@ -173,17 +150,12 @@ export class ConnectWizard {
         currentContext: IKubeconfigEnrichedContext,
         resourceType: ResourceType
     ): Promise<(input: MultiStepInput) => Promise<(input: MultiStepInput) => Promise<(input: MultiStepInput) => Promise<void>>>> {
-        const kubectlClient = await this._binariesUtility.tryGetKubectlAsync();
```

Comment:
💡 for removing `v1` is this path of code removed for specific reasons of seems like this is trying to just make sure `_binariesUtility.tryGetKubectlAsync` if not existing.